build/packaging/rpm/glideinwms.spec

```diff
@@ -251,6 +252,7 @@ install -m 0500 frontend/manageFrontendDowntimes.py $RPM_BUILD_ROOT%{_sbindir}/
 install -m 0500 frontend/stopFrontend.py $RPM_BUILD_ROOT%{_sbindir}/stopFrontend
 install -m 0500 frontend/glideinFrontend.py $RPM_BUILD_ROOT%{_sbindir}/glideinFrontend
 install -m 0500 creation/reconfig_frontend $RPM_BUILD_ROOT%{_sbindir}/reconfig_frontend
+install -m 0500 frontend/renew_proxies.py $RPM_BUILD_ROOT%{_sbindir}/renew_proxies
```

Comment:
Do we realistically expect a sysadmin to run this script by hand?  If so, this is the right directory.  Otherwise, something like `libexec/%{name}` might be better.  

I would prefer some prefix to make it obvious that this command is associated with gwms ... but it seems it's not done elsewhere?  (aside: holy buckets, there's no discernible naming convention here!)